lib/graphql/argument.rb

```diff
@@ -21,6 +21,15 @@ class Argument
 
     ensure_defined(:name, :description, :default_value, :type=, :type)
 
+    def default_value?
+      !!@has_default_value
+    end
+
+    def default_value=(new_default_value)
+      @has_default_value = true
+      @default_value = new_default_value
+    end
```

Comment:
Right 😞 I wasn't sure about that either, I figured if it comes up we can introduce a way to unset it somehow.